src/Pipeline/Transform/PsalmRemapper.php

```
@@ -0,0 +1,127 @@
+
+        $segments     = [];
+        $preferOrigin = $defaultPreferOrigin;
+
+        foreach ($query->segments as $segment) {
+            if ($segment instanceof VerseRef) {
+                [$mapped, $po] = $this->remapVerseRef($segment, $defaultPreferOrigin);
+                $segments[]    = $mapped;
+                $preferOrigin  = $po;
+            } elseif ($segment instanceof VerseRange) {
+                [$from, $poFrom] = $this->remapVerseRef($segment->from, $defaultPreferOrigin);
+                [$to, $poTo]     = $this->remapVerseRef($segment->to, $defaultPreferOrigin);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$poTo'. (undefined)

(UnusedLocalVariable)